scoper.inc.php

```diff
@@ -10,11 +10,14 @@
 
 use Isolated\Symfony\Component\Finder\Finder;
 
+$wp_classes   = json_decode( file_get_contents( 'vendor/sniccwp/php-scoper-wordpress-excludes/generated/exclude-wordpress-classes.json' ) );
+$wp_functions = json_decode( file_get_contents( 'vendor/sniccwp/php-scoper-wordpress-excludes/generated/exclude-wordpress-functions.json' ) );
```

Comment:
I think we can add the constants here too, then this should be good to go!

Reply:
If we're looking to pull from that package -- I'm seeing an empty array at the moment.

<img width="1428" alt="Screen Shot 2022-02-08 at 6 52 38 AM" src="https://user-images.githubusercontent.com/62242/152981845-6e370a68-2c05-403d-a284-e4861d582a50.png">

Reply:
Hmm that's unexpected. Sounds like a bug in that package.

@calvinalkan why is the constants file empty? I'd expect constants like `WP_CONTENT_DIR` and `WP_CONTENT_URL` to be in there.

Reply:
Once we have the answer for that and assuming that's a bug I'll set it up to array merge the rest of the constants

Reply:
Filed an issue at https://github.com/sniccowp/php-scoper-wordpress-excludes/issues/2

Once it's fixed we can do the array merge as you say in a follow-up PR.